spec/openapi.yaml

```diff
@@ -822,6 +822,306 @@ paths:
       deprecated: false
+  /v2-preview/reporting/messages/summary:
+    post:
+      security:
+        - basic_auth: [ ]
+        - hmac_auth: [ ]
+      tags:
+        - Messaging Reports
+      summary: Post summary report
+      description: Create daily report summary containing total number of sent, received and billing units.
+      operationId: PostSummaryReport
+      parameters:
+        - name: start_date
+          in: query
+          description: Start date time for report window. By default, the timezone for this parameter will be taken from the account settings for the account associated with the credentials used to make the request, or the account included in the Account parameter. This can be overridden using the timezone parameter per request. The date must be in ISO8601 format.
+          required: true
+          style: form
+          explode: true
+          schema:
+            type: string
+            example: 2019-12-12T00:00:00.000+11:00
+        - name: end_date
+          in: query
+          description: End date time for report window. By default, the timezone for this parameter will be taken from the account settings for the account associated with the credentials used to make the request, or the account included in the Account parameter. This can be overridden using the timezone parameter per request. The date must be in ISO8601 format.
+          required: true
+          style: form
+          explode: true
+          schema:
+            type: string
+            example: 2019-12-14T00:00:00.000+11:00
+        - name: timezone
+          in: query
+          description: The timezone of the messages to include, using the name of the region.
+          required: true
+          style: form
+          explode: true
+          schema:
+            type: string
+            example: Australia/Sydney
+        - name: direction
+          in: query
+          description: The type of messages to include in the report. Accepted values are `inbound`, `outbound`, and `all` (inbound and outbound)
+          style: form
+          explode: true
+          schema:
+            type: object
+        - name: source
+          in: query
+          description: Filter results by source address.
+          style: form
+          explode: true
+          schema:
+            type: string
+            example: +61555555555
+        - name: destination
+          in: query
+          description: Filter results by destination address.
+          style: form
+          explode: true
+          schema:
+            type: string
+            example: +61555555555
+        - name: metadata_key
+          in: query
+          description: Filter results for messages that include a metadata key. If this parameter is included in the query, then either the metadata_value or metadata_values parameter must also be included to filter the report by the appropriate key for the specified value.
```

Comment:
No `metadata_values` in summary reports but there is a second option to enable the `metadata_key` filter which is to _use `metadata_value` as the `group_by` option_

Reply:
Hey Sam, I think this still applies as `metadata_value` can either be applied as a filter (as you have described currently) or alternatively be used as the `group_by` option.